great_expectations/dataset/pandas_dataset.py

```diff
@@ -750,7 +760,7 @@ def length_is_between(val):
     def expect_column_value_lengths_to_equal(self, column, value,
                                              mostly=None,
                                              result_format=None, include_config=False, catch_exceptions=None, meta=None):
-        return column.map(lambda x: len(x) == value)
+        return column.column.str.len() == value
```

Comment:
I think this is just a typo: -> `return column.str.len() == value`

Reply:
Indeed. Fixed!